consensus/src/quorum_store/network_listener.rs

```diff
@@ -53,27 +50,28 @@ impl NetworkListener {
                             .expect("Failed to send shutdown ack to QuorumStore");
                         break;
                     },
-                    VerifiedEvent::SignedDigestMsg(signed_digest) => {
+                    VerifiedEvent::SignedBatchInfo(signed_digest) => {
                         let cmd = ProofCoordinatorCommand::AppendSignature(*signed_digest);
                         self.proof_coordinator_tx
                             .send(cmd)
                             .await
                             .expect("Could not send signed_digest to proof_coordinator");
                     },
-                    VerifiedEvent::FragmentMsg(fragment) => {
+                    VerifiedEvent::BatchMsg(batch_msg) => {
+                        let batch = batch_msg.unpack();
                         counters::DELIVERED_FRAGMENTS_COUNT.inc();
-                        let idx = fragment.source().to_vec()[0] as usize
+                        let idx = batch.author().to_vec()[0] as usize
```

Comment:
this to_vec()[] looks so ugly, it was there before but anything we can do there? (can it be a method on PeerId?